index.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
+/* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(function (fn) { /* eslint prefer-arrow-callback: 1*/
```

Comment:
I think it's better that fix the lint error for the first error about the require declaration.
Turn off this arrow lint though.